.github/workflows/border_router.yml

```
@@ -60,6 +60,11 @@ jobs:
             otbr_options: "-DOT_TREL=OFF -DOT_DUA=ON -DOT_MLR=ON -DOTBR_COVERAGE=ON -DOT_SRP_SERVER=ON -DOT_ECDSA=ON -DOT_SERVICE=ON -DOTBR_DUA_ROUTING=ON"
             border_routing: 0
             cert_scripts: ./tests/scripts/thread-cert/backbone/*.py
+          - name: "Border Router Avahi Advertising Proxy"
```

```suggestion
          - name: "Border Router (Avahi)"
```

And rename the other one "Border Router (mDNSResponder)" ?